lib/elasticquery/query.rb

```diff
@@ -1,4 +1,4 @@
-require 'active_support/core_ext/hash/deep_merge'
+require 'deep_merge/rails_compat'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.